twx

```diff
@@ -557,6 +643,8 @@ config_info() {
     echo
     echo " > TWX_URL: $TWX_URL"
     echo " > TWX_APPKEY: $TWX_APPKEY"
+    echo " > TWX_USER: $TWX_USER"
+    echo " > TWX_PASSWORD: *******"
```

Comment:
We display appkey in plaintext, can do the same with the password. Even better -- display the first and the last few characters, e.g. `Es###ne`, then you can do the same with the appkey, too, e.g. `A2############55`

Reply:
Done